docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -13,42 +13,54 @@ The sample configuration files below run Semgrep CI on various continuous integr
 ## GitHub Actions
 
 ```yaml
+# Name of this GitHub Actions workflow.
 name: Semgrep
 
 on:
-  # Scan changed files in PRs, block on new issues only (existing issues ignored)
+  # Determine when you want Semgrep to scan your code.
+  # Use as many of the following options as you want.
+  # Option 1: scan changed files in PRs, block on new issues only (existing
```

Comment:
```suggestion
  # Option 1: Scan changed files in PRs, block new issues only (existing
```

Reply:
@efortuna does this mean that: Scan only files changed in PRs. When Semgrep reports a finding it blocks the PR only for the new findings reported from the PR (existing issues in the repository are ignored).

Not sure here, this doubt is applied to all similar sentences in this document.

Reply:
that's correct. Do you have a suggestion for better phrasing? I personally feel like the "on" in the original version is a bit clearer, but if we have a better phrasing overall, I'm down for it.

Reply:
@efortuna It seems it either should be `block` or `put a block on` (but the word of a native is strong):
- https://www.macmillandictionary.com/dictionary/british/put-a-block-on-something
- https://idioms.thefreedictionary.com/block+on

What about: `# Option 1: Let Semgrep report findings only for files changed in PRs. Only findings in the PR become blocking (existing issues in the repository are ignored).`

Reply:
Cool, I changed the sentence entirely, since I realized that we don't even actually "block" on all findings -- it could be "monitor/audit" or "comment"! So I changed the verb to be "report".